src/overlay/OverlayManagerImpl.cpp

```diff
@@ -222,7 +231,11 @@ OverlayManagerImpl::getPreferredPeersFromConfig()
 std::vector<PeerRecord>
 OverlayManagerImpl::getPeersToConnectTo(int maxNum)
 {
-    const int batchSize = std::max(20, maxNum);
+    // don't connect to too many peers at once
+    maxNum = std::min(maxNum, 50);
+
+    // batch is how many peers to load from the database every time
+    const int batchSize = std::max(50, maxNum);
```

Comment:
I think it will be 50 every time now as it is basically std::max(50, std::min(any, 50))

Reply:
yes this is correct, but because they are different context I prefer to keep the formulas in code as they don't need to be both 50